docs/11-layout-engine.md

```
@@ -0,0 +1,975 @@
+
+**"LLM이 좌표를 만든다"**를 포기하고, **"LLM은 구조를 선언하고 엔진이 좌표를 만든다"** 로 전환한다. 이것이 본 문서가 제안하는 방향이다.
+
+---
+
+## 2. 설계 원칙
+
+### 2.1 구조 선언형 (semantic-first)
+
+LLM 출력은 **"노드가 뭐고, 어떻게 이어져 있는가"** 만 표현한다. 좌표·크기는 엔진이 결정.
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)